packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -407,4 +401,8 @@ export default defineComponent({
 
 <style lang="scss">
 @import "variables";
+
+.va-time-input__side-button {
+  pointer-events: var(--va-time-input-side-button-pointer-events);
```

Comment:
Hey, @Udaay. We use CSS variables, so it is easier for user to change component look. Do you think there is a case when user need to change pointer-events behavior? 

The perfect examples of using css variables to configure border-radius, border-width, paddings, margins, etc.

What I don't like is to have css variables, that for 99% will not be changed. Like display (which is set to flex and whole component breaks if display is changed), position (which must be relative otherwise everything breaks).

We actually do not follow this rule completely, we have a lot of old stuff which was converted to css variables by a script. And I hope in future we will have less meaningless variables. 

If you find pointer-events css variable helpful here, let's have it, but if not please remove the css variable. Thanks.

Reply:
Yes , I got your point and whole idea behind the css variables so that user can configure some specific styles.
Will remove this variable and add directly.